fluid/object_detection/train.py

```diff
@@ -12,46 +12,35 @@
-add_arg('use_gpu', bool, True, "Whether use GPU.")
-add_arg('data_dir', str, './data/COCO17', "Root path of data")
-add_arg('train_file_list', str, 'annotations/instances_train2017.json',
-        "train file list")
-add_arg('val_file_list', str, 'annotations/instances_val2017.json',
-        "vaild file list")
-add_arg('model_save_dir', str, 'model_COCO17', "where to save model")
-
-add_arg('dataset', str, 'coco', "coco or pascalvoc")
-add_arg(
-    'is_toy', int, 0,
-    "Is Toy for quick debug, 0 means using all data, while n means using only n sample"
-)
-add_arg('label_file', str, 'label_list',
-        "Lable file which lists all label name")
-add_arg('apply_distort', bool, True, "Whether apply distort")
-add_arg('apply_expand', bool, False, "Whether appley expand")
-add_arg('resize_h', int, 300, "resize image size")
-add_arg('resize_w', int, 300, "resize image size")
-add_arg('mean_value_B', float, 127.5,
-        "mean value which will be subtracted")  #123.68
-add_arg('mean_value_G', float, 127.5,
-        "mean value which will be subtracted")  #116.78
-add_arg('mean_value_R', float, 127.5,
-        "mean value which will be subtracted")  #103.94
-
-
-def train(args,
-          train_file_list,
-          val_file_list,
-          data_args,
-          learning_rate,
-          batch_size,
-          num_passes,
-          model_save_dir='model',
-          init_model_path=None):
+# yapf: disable
+add_arg('learning_rate',    float, 0.001,     "Learning rate.")
+add_arg('batch_size',       int,   32,        "Minibatch size.")
+add_arg('num_passes',       int,   25,        "Epoch number.")
+add_arg('parallel',         bool,  True,      "Whether use parallel training.")
+add_arg('use_gpu',          bool,  True,      "Whether use GPU.")
+add_arg('dataset',          str, 'pascalvoc', "coco or pascalvoc.")
+add_arg('model_save_dir',   str, 'model',     "The path to save model.")
+add_arg('pretrained_model', str, 'pretrained/ssd_mobilenet_v1_coco/', "The init model path.")
+add_arg('apply_distort',    bool, True,   "Whether apply distort")
+add_arg('apply_expand',     bool, False,  "Whether appley expand")
+add_arg('resize_h',         int,  300,    "resize image size")
+add_arg('resize_w',         int,  300,    "resize image size")
+add_arg('mean_value_B',     float, 127.5, "mean value which will be subtracted")  #123.68
+add_arg('mean_value_G',     float, 127.5, "mean value which will be subtracted")  #116.78
+add_arg('mean_value_R',     float, 127.5, "mean value which will be subtracted")  #103.94
+add_arg('is_toy',           int, 0, "Toy for quick debug, 0 means using all data, while n means using only n sample")
+# yapf: disable
+
+
+def parallel_do(args,
+                train_file_list,
+                val_file_list,
+                data_args,
+                learning_rate,
+                batch_size,
+                num_passes,
+                model_save_dir,
+                pretrained_model=None):
```

Comment:
`parallel_do` need have an argument to define using `nccl` or not. The default value does not use `nccl`.

Reply:
Done.